examples/xml_test/Doxyfile.xml

```diff
@@ -0,0 +1,398 @@
+<?xml version='1.0' encoding='UTF-8' standalone='no'?>
```

Comment:
Why introducing in example only xml content? My expectation is that the example folder has a real example which shows how the tool works with the doxygen xml output in a subfolder like in a standard doxygen configuration.